src/Android/Tiles/AutofillTileService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Service.QuickSettings;
+using Bit.App.Resources;
+using Bit.Droid.Accessibility;
+using Java.Lang;
+
+namespace Bit.Droid.Tile
+{
+    [Service(Permission = Android.Manifest.Permission.BindQuickSettingsTile, Label = "@string/ScanAndFill",
+        Icon = "@drawable/shield")]
+    [IntentFilter(new string[] { ActionQsTile })]
+    [Register("com.x8bit.bitwarden.AutofillTileService")]
+    public class AutofillTileService : TileService
+    {
+        public override void OnTileAdded()
+        {
+            base.OnTileAdded();
+            AccessibilityHelpers.IsAutofillTileAdded = true;
```

Comment:
Does this only work whenever you first turn on the tile? Is this function called again when the device restarts, for example?

Reply:
I thought it did but I fooled myself apparently.  Persisting in storage now.